dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -42,27 +42,48 @@ ENV BEAT_SETUID_AS={{ .user }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do microdnf update -y && microdnf install -y shadow-utils jq &&  microdnf clean all && exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
 {{- else }}
-# Installing jq needs to be installed after epel-release and cannot be in the same yum install command.
-RUN case $(arch) in aarch64) YUM_FLAGS="-x bind-license";; esac; \
-    for iter in {1..10}; do \
-        yum update -y $YUM_FLAGS && \
-        yum install -y epel-release &&  \
-        yum update -y $YUM_FLAGS && \
-        yum install -y jq && \
-        yum clean all && \
-        exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; \
+
+RUN for iter in {1..10}; do \
+        apt update -y && \
+        apt install --no-install-recommends --yes jq && \
+        apt clean all && \
+        exit_code=0 && break || exit_code=$? && echo "apt error: retry $iter in 10s" && sleep 10; \
     done; \
     (exit $exit_code)
 {{- end }}
 
+## Changes Notes: To migratre from CentOS to Ubuntu, the following changes are needed:
```

Comment:
Those are all packages migrated from CentOs to Ubuntu related

Reply:
Is this temporarily for discussion or you plane to level this in here?

Reply:
Hello @ruflin, this is for temporary discussion, I'll try to find where I add them to a changelog for future notes. 😸